src/inspect_flow/_types/generated.py

```diff
@@ -731,7 +731,7 @@ class FlowTask:
     epochs: Optional[Union[int, FEpochs, FlowEpochsDict, FlowEpochs]] = None
     """Epochs to repeat samples for and optional score reducer function(s) used to combine sample scores (defaults to "mean")"""
     fail_on_error: Optional[Union[bool, float]] = None
-    """`True` to fail on first sample error(default); `False` to never fail on sample errors; Value between 0 and 1 to fail if a proportion of total samples fails. Value greater than 1 to fail eval if a count of samples fails."""
+    """`True` to fail on first sample error (default); `False` to never fail on sample errors; Value between 0 and 1 to fail if a proportion of total samples fails. Value greater than 1 to fail eval if a count of samples fails."""
```

Comment:
This is generated from flow_types, so will need to fix there. If you use the pre commit hooks it will catch this.